### PR TITLE
fix(arc): Fix handling spaced data name

### PR DESCRIPTION
**Pull request by netil**
## Issue

#1168

## Details

Replace arc data names having space(or special chars).

`'data 1 2' --> 'data-1-2'`

### CODE REVIEW

File: spec/interactions/interaction-spec.js [hunk 1]
```diff
@@ -1012,4 +1012,32 @@ describe("INTERACTION", () => {
...
+				chart.internal.setOverOut(true, {
+					id: "catégorie 1"
+				});
+
+				chart.internal.setOverOut(true, {
+					id: "catégorie 2 2"
+				});
+
+				expect(true).to.be.ok;
+			} catch(e) {}
```
Comment (stof) [inline spec/interactions/interaction-spec.js]: wouldn't this try/catch hide the fact that there is an error, and so not make the test fail ?

Comment (netil) [reply spec/interactions/interaction-spec.js]: you're right. will pass through even has an error.
I'll be updating that. Thanks to notice!

**Comment by stof** [comment]
@netil is there any chance to have this released as a 1.11.2 release ? Otherwise, is there an ETA for 1.12 reaching stable release ?

**Comment by netil** [comment]
@stof, the next release `1.12.0` will be out in this month. Probably, at the 3rd or 4th week of March.
Try testing the `@next` tagged version meanwhile.

**Pull request merged**